src/pages/chat.tsx

```diff
@@ -1,18 +1,49 @@
+//@ts-nocheck
 import type { NextPage } from 'next';
 import Image from 'next/image';
 import avatarMale from '../images/avatar/male.png';
 import Chat from '../components/chat';
-import ChatBox from '../components/chatbox';
+import { useState, useRef } from 'react';
+import { useSession, signOut } from 'next-auth/react';
+
+// Temporarily store messages of a user
+const messages = [];
 
 const ChatPage: NextPage = (props): JSX.Element => {
+    const { data: session } = useSession();
+    const dummy = useRef();
+    const [chatText, setChatText] = useState('');
+
+    if (!session) return <p>Access Denied.</p>;
+
+    const userTyping = (event: any) => {
```

Comment:
If you put the message and submit button into a form, you don't have to do any key checking.

Reply:
I got this implemented in the latest commit.